eng/pipelines/runtime.yml

```diff
@@ -521,14 +521,13 @@ extends:
           alwaysRun: ${{ variables.isRollingBuild }}
           extraBuildArgs: /p:AotHostArchitecture=x64 /p:AotHostOS=$(_hostedOS)
 
-      # Build and Smoke Tests only - Wasm Threading Legs
+      # Wasm Library tests - Threading Legs
       - template: /eng/pipelines/common/templates/wasm-library-tests.yml
         parameters:
           platforms:
             - browser_wasm
-          nameSuffix: _Threading_Smoke
+          nameSuffix: _Threading
           extraBuildArgs: /p:MonoWasmBuildVariant=multithread /p:_WasmPThreadPoolSize=8 /p:AotHostArchitecture=x64 /p:AotHostOS=$(_hostedOS)
-          shouldRunSmokeOnly: true
```

Comment:
Maybe it should stay smoke on default pipeline ? And full on runtime-wasm. I'm not sure if we are not low on CI resources right now.

Reply:
@radical, I think you can help with the decision

Reply:
Yeah, we should keep smoke tests on `runtime.yml` for now, till we at least know that the tests are stable.